src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
```

Comment:
The class declaration should be followed by an opening brace '{' on the same line according to Java code conventions.

Reply:
Thanks, fixed.